sections/contact-form.liquid

```diff
@@ -16,7 +16,11 @@
 
 <div class="color-{{ section.settings.color_scheme }} gradient">
   <div class="contact page-width page-width--narrow section-{{ section.id }}-padding">
-    <h2 class="title title-wrapper--no-top-margin {{ section.settings.heading_size }}">{{ section.settings.heading | escape }}</h2>
+    {%- if section.settings.heading != blank -%}
+      <h2 class="title title-wrapper--no-top-margin {{ section.settings.heading_size }}">{{ section.settings.heading | escape }}</h2>
+    {%- else -%}
+      <h2 class="visually-hidden">{{ 'templates.contact.form.title' | t }}</h2>
```

Comment:
@martinamarien Is `templates.contact.form` the expected place for this to live? Being a a11y specific string I wasn't sure if it should go in the `accessibility` object instead (seen here https://github.com/Shopify/dawn/pull/1515/commits/d7e03f652fb288b8e465a8c2750e61140709389c)

Reply:
Templates is perfectly fine. 
I personally view the `a11y` strings are ones that are repurposed elsewhere in the theme. 
For example: preview, next, close, skip, etc.